functions.php

```diff
@@ -776,7 +783,7 @@ function export_graphs(&$export, $export_path) {
 
 			$exported++;
 
-			if ($exported >= $export['graph_max']) {
+			if ($exported >= $export['graph_max'] && $export['graph_max'] > 0) {
```

Comment:
This seems OK, though it could be shorted to `$export['graph_max'] && $exported >= $export['graph_max']`

Reply:
Was intentionally done this way to be a more visible change to original code.  Preference?